test/Microsoft.Sbom.Targets.Tests/AbstractGenerateSBomTaskInputTests.cs

```diff
@@ -249,22 +282,32 @@ public void Sbom_Generation_Succeeds_For_Invalid_Verbosity()
 
         // Assert
         Assert.IsTrue(result);
+#if NET472
+        Assert.IsTrue(this.messages.Any(msg => pattern.IsMatch(msg.Message)));
+#else
         Assert.IsTrue(pattern.IsMatch(output));
+#endif
     }
 
+#if !NET472
     /// <summary>
     /// Test to ensure GenerateSbom correctly parses and provides each EventLevel verbosity
     /// values to the SBOM API.
     /// </summary>
     [TestMethod]
-    [DataRow("FATAL", "Fatal")]
-    [DataRow("information", "Information")]
-    [DataRow("vErBose", "Verbose")]
-    [DataRow("Warning", "Warning")]
-    [DataRow("eRRor", "Error")]
-    [DataRow("Debug", "Verbose")]
-    public void Sbom_Generation_Assigns_Correct_Verbosity_IgnoreCase(string inputVerbosity, string mappedVerbosity)
+    [DataRow("FATAL", "Fatal", false)]
+    [DataRow("information", "Information", true)]
+    [DataRow("vErBose", "Verbose", true)]
+    [DataRow("Warning", "Warning", false)]
+    [DataRow("eRRor", "Error", false)]
+    [DataRow("DeBug", "Verbose", true)]
+    public void Sbom_Generation_Assigns_Correct_Verbosity_IgnoreCase(string inputVerbosity, string mappedVerbosity, bool messageShouldBeLogged)
     {
+        if (!messageShouldBeLogged)
```

Comment:
I would add a comment saying this is specifically happening for the SBOM API

Reply:
This test method already has a comment saying that this is only for the SBOM SPI [here](https://github.com/microsoft/sbom-tool/pull/613/files/8b43205780abbc145580a80ab2dec0f32db9d92c#diff-f861882562cf5a54bdf65ec173d96298d6bff37e303af42262559debe85c5c46R295).